--- edgar-0.93/makefile.orig	2011-09-09 18:59:03.000000000 +0200
+++ edgar-0.93/makefile	2011-11-23 22:50:49.474026241 +0100
@@ -22,7 +22,7 @@
 CXX       = gcc
 
 PREFIX = $(DESTDIR)/usr
-BIN_DIR = $(PREFIX)/games/
+BIN_DIR = $(PREFIX)/bin/
 DOC_DIR = $(PREFIX)/share/doc/$(PROG)/
 ICON_DIR = $(PREFIX)/share/icons/hicolor/
 DESKTOP_DIR = $(PREFIX)/share/applications/
@@ -35,13 +35,8 @@
 DATA_DIR = $(PREFIX)/share/games/edgar/
 endif
 
-ifeq ($(DEV),1)
-CFLAGS = -Wall -g -pedantic -Werror -DVERSION=$(VERSION) -DRELEASE=$(RELEASE) -DDEV=$(DEV) -DINSTALL_PATH=\"$(DATA_DIR)\" -DLOCALE_DIR=\"$(LOCALE_DIR)\" -DPAK_FILE=\"$(PAK_FILE)\"
-#CFLAGS = -Wall -O2 -pedantic -DVERSION=$(VERSION) -DRELEASE=$(RELEASE) -DDEV=$(DEV) -DINSTALL_PATH=\"$(DATA_DIR)\" -DLOCALE_DIR=\"$(LOCALE_DIR)\" -DPAK_FILE=\"$(PAK_FILE)\"
-else
-CFLAGS = -Wall -O2 -pedantic -DVERSION=$(VERSION) -DRELEASE=$(RELEASE) -DDEV=$(DEV) -DINSTALL_PATH=\"$(DATA_DIR)\" -DLOCALE_DIR=\"$(LOCALE_DIR)\" -DPAK_FILE=\"$(PAK_FILE)\"
+CFLAGS = $(OPTFLAGS) -DVERSION=$(VERSION) -DRELEASE=$(RELEASE) -DDEV=$(DEV) -DINSTALL_PATH=\"$(DATA_DIR)\" -DLOCALE_DIR=\"$(LOCALE_DIR)\" -DPAK_FILE=\"$(PAK_FILE)\"
 #CFLAGS = -Wall -Os -pedantic -DVERSION=$(VERSION) -DRELEASE=$(RELEASE) -DDEV=$(DEV) -DINSTALL_PATH=\"$(DATA_DIR)\" -DLOCALE_DIR=\"$(LOCALE_DIR)\" -DPAK_FILE=\"$(PAK_FILE)\"
-endif
 
 ifeq ($(OS),Windows_NT)
 LFLAGS = `sdl-config --libs` -lSDL -lSDL_image -lSDL_mixer -lSDL_ttf -lSDL_net -lz -lintl -lm
@@ -101,7 +96,7 @@
 all : $(PROG) makefile.dep $(ED_PROG) $(PAK_PROG) $(LOCALE_MO) $(PO_PROG)
 
 makefile.dep : src/*/*.h src/*.h
-	for i in src/*.c src/*/*.c; do gcc -MM "$${i}"; done > $@
+	for i in src/*.c src/*/*.c; do $(CXX) -MM "$${i}"; done > $@
 
 # compiling other source files.
 %.o:
@@ -112,23 +107,23 @@
 
 # linking the program.
 $(PROG): $(MAIN_OBJS) $(CORE_OBJS)
-	$(CXX) $(MAIN_OBJS) $(CORE_OBJS) -o $(PROG) $(LFLAGS)
+	$(CXX) $(LDFLAGS) $(MAIN_OBJS) $(CORE_OBJS) -o $(PROG) $(LFLAGS)
 	
 # linking the program.
 $(ED_PROG): $(EDIT_OBJS) $(CORE_OBJS)
-	$(CXX) $(EDIT_OBJS) $(CORE_OBJS) -o $(ED_PROG) $(LFLAGS)
+	$(CXX) $(LDFLAGS) $(EDIT_OBJS) $(CORE_OBJS) -o $(ED_PROG) $(LFLAGS)
 
 # linking the program.
 $(PAK_PROG): $(PAK_OBJS)
-	$(CXX) $(PAK_OBJS) -o $(PAK_PROG) $(LFLAGS)
+	$(CXX) $(LDFLAGS) $(PAK_OBJS) -o $(PAK_PROG) $(LFLAGS)
 	
 # linking the program.
 $(PO_PROG): $(PO_OBJS)
-	$(CXX) $(PO_OBJS) -o $(PO_PROG) $(LFLAGS)
+	$(CXX) $(LDFLAGS) $(PO_OBJS) -o $(PO_PROG) $(LFLAGS)
 
 # linking the program.
 $(TILE_PROG): $(TILE_OBJS)
-	$(CXX) $(TILE_OBJS) -o $(TILE_PROG) $(LFLAGS)
+	$(CXX) $(LDFLAGS) $(TILE_OBJS) -o $(TILE_PROG) $(LFLAGS)
 
 # cleaning everything that can be automatically recreated with "make".
 clean:
