--- edgar-0.60/makefile~	2010-06-13 14:35:59.916577276 +0200
+++ edgar-0.60/makefile	2010-06-13 14:37:59.439903415 +0200
@@ -39,7 +39,7 @@
 ifeq ($(OS),Windows_NT)
 LFLAGS = `sdl-config --libs` -lSDL -lSDL_image -lSDL_mixer -lSDL_ttf -lSDL_net -lz -llibintl -lm
 else
-LFLAGS = `sdl-config --libs` -lSDL -lSDL_image -lSDL_mixer -lSDL_ttf -lSDL_net -lz
+LFLAGS = `sdl-config --libs` -lSDL -lSDL_image -lSDL_mixer -lSDL_ttf -lSDL_net -lz -lm
 endif
 
 TILE_OBJS  = tile_creator.o save_png.o
